### DISCUSSION TITLE
Graphql loaded draft pages with draft: false specified

### DISCUSSION THREAD

**Discussion by Stupidism** [payloadcms/payload#1295] [Q&A]
It looks a bug to me that I can query out draft pages with following query in the official demo page: 
https://demo.payloadcms.com/api/graphql-playground

```
# Write your query or mutation here
query getPages {
  Pages(draft: false) {
    docs {
      slug
      _status
    }
  }
}
```

<img width="1319" alt="image" src="https://user-images.githubusercontent.com/5801015/197689102-8736a606-f481-4862-a981-fa075262010d.png">

I found out I need to add one more filter like this, but I'm still confused

```
query getPages {
  Pages(draft: false, where:{ _status:{ equals: published } }) {
    docs {
      slug
      _status
    }
  }
}
```

**Comment by PatrikKozak** [comment]
Related and closed with #1566
